presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -209,6 +211,19 @@ public void disableRubix(Configuration configuration)
         setCacheKey(configuration, "rubix_disabled");
     }
 
+    public enum ConfigurationOwner {
+        PRESTO,
+        RUBIX,
+    }
+
+    public static ConfigurationOwner getConfigurationOwner(Configuration configuration)
```

Comment:
alternatively we could extend extension point with `ConfigurationOwner`, but I'm fine with this too

Reply:
Yeah - I think it will be cleaner. I will change the code.